Enterprise-CMCS/macpro-mdct-mcr
# [MDCT-2313] Even more MLR redlines!

### Description
Fix a few small errors with the PDF page.

Currently pointing to "mlr-statusing-2313" because it relies on work in that branch.

### Related ticket(s)
MDCT-2428

---
### How to test
1. Create and fill a new MLR submission
2. Verify AC in ticket has been met.

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
